### PR TITLE
ES6 syntax and modules

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
in this capston having the original functionalities I applied the following changes:
1- used ES6 syntax
2- used Modules and separated codes into multiple JS files in Module directory
3- add Date & time so that the users can have the time displayed

thanks you

### CODE REVIEW

File: modules/bookList.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// Book Class: Represents a Book
+
```
Comment (omar-labana) [inline modules/bookList.js]: - [x] According to the project requirements, you should omit the usage of the function keyword to declare functions across the code base. instead please use the arrow function declaration.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @sediqullahbadakhsh, 

Excellent work so far! 👌
You have implemented all the required features without any bugs or errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_N/A_ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/bookList.js (hunk 1 above)
Comment (sediqullahbadakhsh) [reply modules/bookList.js]: Thanks Ill do that

File: modules/bookList.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//   const books = getBooks();
+
+//   books.forEach((book, index) => {
+//     if (book.author === author) {
+//       books.splice(index, 1);
+//     }
+//   });
+
+//   localStorage.setItem("books", JSON.stringify(books));
+// });
```
Comment (oloomoses) [inline modules/bookList.js]: - [x] [**Optional**] Kindly remove the commented pieces of code because they make your work look messy.

**Review by oloomoses — APPROVED** [review]
Hi @sediqullahbadakhsh, 

Well done on implementing the suggested changes. :muscle: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/bookList.js (hunk 2 above)
Comment (sediqullahbadakhsh) [reply modules/bookList.js]: oh, okay, sure thanks, I really forgot to do it. thank you

**Pull request merged**